solutions/swb-ui/ui/scripts/image-deploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
nit: #!/usr/bin/env bash

https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash

Reply:
Updated.